### DISCUSSION TITLE
Smartmeterwerte über Homeassistant bereitstellen?

### DISCUSSION THREAD

**Discussion by lemonfunk** [hoylabs/OpenDTU-OnBattery#1053] [Q&A]
Hallo,
Ich bin grad dabei ein BKW mit SmartSolar 100/20, 48VPufferakku, HM700 und OpenDTU aufzusetzen. Zuerst hatte ich gedacht, dass ich mich um die Regelung selbst kümmern muss, bis ich gesehen habe, dass es die OpenDTU Onbattery Firmware gibt ;)

Nun zu meiner Frage: Ich habe ein Elgris Smartmeter am Netzanschlusspunkt und habe die entsprechenden Werte auch in Homeassistant zur Verfügung. Mir ist noch nicht ganz klar, Welche Optionen OpenDTU Onbattery alles für eine Smartmeteranbindung bietet. Wie könnte ich OpenDTU Onbattery in diesem Fall konfigurieren um auf die notwendigen Werte zuzugreifen?
Danke & Gruß,
Benjamin

**Comment by spcqike** [comment]
Ich vermute am einfachsten per MQTT. Publishe den Wert in deinem gewünschten und sinnvollen Intervall. Und lass die openDTU onBattery per MQTT darauf lauschen.

**Comment by schlimmchen** [comment]
Ich stimme @spcqike zu. Seine Unterstellung ist, dass die Daten schon in deinem MQTT Broker zur Verfügung stehen, wenn sie schon in Homeassistant sind. Betreibst du diesen MQTT Broker selbst? Hast du deine Homeassistent Installation daraufhin konfiguriert? Hast du mal mit dem MQTT Explorer auf deinen Broker geschaut? Da wirst du das Topic, in das dein Smartmeter die Werte veröffentlich recht leicht finden können.

Andernfalls brauchen wir mehr Infos über deinen Smartmeter und dessen Anbindung (genaues Modell, verbindung mit deinem Netzwerk, oder ob du den mit einem Hichi oder etwas ähnlichem über die IR-Schnittstelle ausließt, etc.).
